addons/base_vat/models/res_partner.py

```diff
@@ -189,6 +192,15 @@ def check_vat_ch(self, vat):
             return check == int(num[8])
         return False
 
+     def check_vat_cl(self, vat):
+        return stdnum.util.get_cc_module('cl','vat').is_valid(vat)
+
+    def check_vat_co(self, vat):
+        #VAT validation in stdnum library need VAT without 'CO' in starting so remove if given
```

Comment:
@jva-odoo Normally it will fail and then the method calling this will test without CO afterwards anyways.  I also don't understand why it would be different for Chile as for Colombia.  Does stdnum work differently there?

Reply:
@jco-odoo Yes, for Chil stdnum it's self remove CL for CO it's not.
Yes, it's work without it because as you say when test fails then odoo check again without country code.
So you like to remove this.?

Reply:
According to that logic, we can indeed keep it even if not strictly necessary

Reply:
ok then I remove it because it's not strictly necessary